<launch>
  <!-- Launch server separatly (better) -->
  <!-- <include file="$(find vive_ros)/launch/server_vr.launch"> -->
    <!-- <arg name="moveit_warehouse_database_path" value="$(arg db_path)"/> -->
  <!-- </include> -->

  <node name="vive_pose" pkg="vive_ros" type="vive_pose"/>

  <node name="panda_dual_simultaneous_control" pkg="moveit_vive" type="panda_dual_simultaneous_control" output="screen">
    <!-- <param name="arm_id" type="str" value="panda2"/> -->
    <!-- <param name="controller_id" type="str" value="left"/> -->
  </node>

</launch>
 